Cargo.toml

```diff
@@ -32,7 +32,7 @@ default = ["cli", "repl"]
 cli = ["bdk/key-value-db", "clap", "dirs-next", "env_logger"]
 repl = ["regex", "rustyline"]
 electrum = ["bdk/electrum"]
-esplora = ["bdk/esplora"]
+esplora = ["bdk/use-esplora-ureq"]
```

Comment:
I'm trying to use this to test bitcoindevkit/bdk#429: to get bdk-cli to work in the browser we need a way to enable `bdk/use-esplora-reqwest`. I would suggest maybe adding another feature `esplora-reqwest` that we can use from wasm32

Reply:
Or maybe even better: `async-esplora`

Reply:
I think this PR may be blocked until @tcharding and @LLFourn  work out things out with https://github.com/bitcoindevkit/bdk/pull/433.

Reply:
Yes. I am not really sure how big the httpclient fix would be. If it takes much longer,  can we consider merging https://github.com/bitcoindevkit/bdk/pull/430 in the mean time? #36  is also pending on this one.

And I think that will fix @afilini 's issue too.

Reply:
In the meantime I made this branch just to test the playground: https://github.com/afilini/bdk-cli/tree/update-with-reqwest